README.md

```diff
@@ -164,7 +164,29 @@ public class WebServerSpec {
     }
 }
 ```
-          
+
+#### Automatic generation of CRDs
+
+To automatically generate CRD manifests from your annotated Custom Resource classes, you only need
+to add the following dependency to your project:
+
+```xml
+<dependency>
+  <groupId>io.fabric8</groupId>
+  <artifactId>crd-generator</artifactId>
+  <scope>compile</scope>
+</dependency>
+```
+
+The CRD will be generated in `target/classes/META-INF/fabric8` (or
+in `target/test-classes/META-INF/fabric8`, if you use the `test` scope) with the CRD name suffixed
```

Comment:
I'm wondering if this is the best place to put the generated files... Why here? Why not in resources or even in the kubernetes directory next to the generated deployment yaml?

Reply:
Because that's where fabric8 tools have traditionally put their generated files. If that doesn't work for you, please open an issue in the client project.

Reply:
See fabric8io/kubernetes-client#2842.

Reply:
I see you already did, thanks!